common/src/main/java/com/google/i18n/addressinput/common/FormOptions.java

```diff
@@ -22,7 +22,6 @@
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
-import java.util.Locale;
```

Comment:
What did this? Sure, it's not used in this class, so this seems to work, but it is still there in the source file...